### PR TITLE
New instance template: ask to install filton or antibot-proxy

### PR DESCRIPTION
**Pull request by dalf**

### CODE REVIEW

**Comment by unixfox** [comment]
I don't think we should require that, there could be some valid reasons that one doesn't want to install that kinf of antibot software.

The best way is to notify through the GitHub template that one can install this kind of protection in order to reduce the amount of bots. It's already the case but maybe we should instead require to read the anti bot explanation part?

**Comment by dalf** [comment]
> there could be some valid reasons that one doesn't want to install that kinf of antibot software.

Do you have some use case example ? 

> I don't think we should require that

Until now, my idea is to not include an instance without an antibot software, but clearly nowhere it is said that this is requirement, so it can be frustrating to have a "fence" out of nowhere asking to install filtron.

> It's already the case but maybe we should instead require to read the anti bot explanation part?

How to do that ?
The only place I see, it add an additional message in README.md

**Comment by unixfox** [comment]
> > there could be some valid reasons that one doesn't want to install that kinf of antibot software.
> 
> Do you have some use case example ?

The filtron rules from searx-docker can sometimes misbehave and block legitimate requests that we don't know until we discover about it, like the one from Firefox preview (see https://github.com/searx/searx-docker/issues/48).   
I know it's a controversial example but if a maintainer really wants to publish publicly a vanilla Searx instance without any protection then he should be free to do so.

> 
> > I don't think we should require that
> 
> Until now, my idea is to not include an instance without an antibot software, but clearly nowhere it is said that this is requirement, so it can be frustrating to have a "fence" out of nowhere asking to install filtron.

I do understand your point of view, but we should only recommend that not forcing the maintainer to install the protection.

> > It's already the case but maybe we should instead require to read the anti bot explanation part?
> 
> How to do that ?
> The only place I see, it add an additional message in README.md

Require to acknowledge that the maintainer read the `Be sure to have configured filtron` part and understand the consequences of not having it installed before submitting the instance.

**Comment by dalf** [comment]
I've updated the template.

File: .github/ISSUE_TEMPLATE/add-instance.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ assignees: ''
 * [ ] It is my instance.
 * [ ] I give the right to searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
 
+Pick one:
+* [ ] I have install [filtron](https://github.com/asciimoo/filtron/), [antibot-proxy](https://github.com/unixfox/antibot-proxy) or another bot protection.
+* [ ] I do recognize that without a bot protection, my instance will be spam with bots.
```
Comment (unixfox) [inline .github/ISSUE_TEMPLATE/add-instance.md:16]: I would say in a scarier way:
> I do recognize that without a bot protection, my instance will be flooded by bots and could have some of its engines stop working (an example is Google).

File: .github/ISSUE_TEMPLATE/add-instance.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ assignees: ''
 * [ ] It is my instance.
 * [ ] I give the right to searx.space to check my instance (every 3 hours for the response times, every 24 hours for the other tests).
 
+Pick one:
+* [ ] I have install [filtron](https://github.com/asciimoo/filtron/), [antibot-proxy](https://github.com/unixfox/antibot-proxy) or another bot protection.
```
Comment (unixfox) [inline .github/ISSUE_TEMPLATE/add-instance.md:15]: I'm not a native English speaker but I think `Ì have install` is grammatically incorrect and should be instead `Ì have installed`.

**Comment by dalf** [comment]
Updated.

**Review by unixfox — APPROVED** [review]

**Pull request merged**